app.js

```diff
@@ -150,19 +150,28 @@ function multiplyAnswer(a,b){
 function multiplyArray(testArray){ //eslint-disable-line
   var arrayValue;
   var calculatedMultiply = 1;
-  if (testArray){
+  if(testArray.length > 0){
     for(var i = 0;i < testArray.length; i++){
-      arrayValue = testArray[i];
-      calculatedMultiply = multiplyAnswer(arrayValue,calculatedMultiply);
-      console.log('calculatedMultiply :: ' + calculatedMultiply);
-      console.log('The numbers ' + testArray + ' have a product of ' + calculatedMultiply + '.');
+      if (!isNaN(testArray[i] && testArray[i] !== null)){
+        arrayValue = testArray[i];
+        calculatedMultiply = multiplyAnswer(arrayValue,calculatedMultiply);
+        console.log('calculatedMultiply :: ' + calculatedMultiply);
+        console.log('The numbers ' + testArray + ' have a product of ' + calculatedMultiply + '.');
+      }
+      else{
+        console.log('You broke it! put numbers in your array!');
+        return 'You broke it! put numbers in your array!';
+      }
     }
+    return[
+      calculatedMultiply
+      ,'The numbers ' + testArray + ' have a product of ' + calculatedMultiply + '.'
+    ];
+  }
+  else{
+    console.log('You broke it! put numbers in your array!');
+    return 'You broke it! put numbers in your array!';
```

Comment:
For future reference, this might be a good place to use `throw`. If someone is expecting an array to be returned, they'll be surprised by a non-array result. Especially when strings can act array-ish!